Reference updates (#75)
crate/cloud-reference: docs/overview.rst

     recently restarted a node. Our support team is already notified and
     investigating the issue.

* **Region**: Name of the region where the cluster is deployed.

* **Plan**: This shows what subscription plan the cluster is running on. For
  more information on our plans, see the documentation on
  `subscription plans`_.

> * **CPU metrics**: Current CPU utilization of the cluster.

Reviewer: ```suggestion
* **CPU metrics**: Average CPU utilization on average per node. The sparkline shows the trend for the last hour.
```

* **Number of nodes**: Number of nodes in the cluster.

> * **RAM metric**: Amount of memory of your cluster.

Reviewer: ```suggestion
* **RAM metric**: Percentage of ram used in each node on average. The sparkline shows the trend for the last hour.
```

> * **Storage metrics**: Used and overall storage of the cluster.

Reviewer: ```suggestion
* **Storage metrics**: Used and overall storage of the cluster. The sparkline shows the trend for the last hour.
```

Next Steps
~~~~~~~~~~

* **Import Data**: Import some data into your cluster using our data import 
  tool.

> * **See my backups**: CrateDB Cloud clusters can now be cloned to a new 
>   cluster from any backup.

Reviewer: I think this was a a pretty good explanation previously? Maybe add the clone sentence at the end?

```suggestion
* **See my backups**: The "see my backups" will take you to the Backups tab, where 
  you can see all your backups. CrateDB Cloud clusters can now be cloned to a new 
  cluster from any backup.
```
